packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -402,14 +404,29 @@ function FontLibraryProvider( { children } ) {
 	};
 
 	const activateCustomFontFamilies = ( fontsToAdd ) => {
-		// Merge the existing custom fonts with the new fonts.
+		// Removes the id from the families and faces to avoid saving that to global styles post content.
```

Comment:
I think we could explain here _why_ we want to avoid saving to global styles post content.

Reply:
Agreed, should explain.

Reply:
Although I think we can do that in a follow up.